src/clarity-angular/button-group/index.ts

```diff
@@ -6,8 +6,10 @@
 import { Type } from "@angular/core";
 import {Button} from "./button";
 import {ButtonGroup} from "./button-group";
+import {LoadingButton} from "./button-loading";
 
 export const BUTTON_GROUP_DIRECTIVES: Type<any>[] = [
     ButtonGroup,
-    Button
+    Button,
+    LoadingButton
```

Comment:
This is not really part of the button group, so I'd rather you exported a separate array to keep things flexible.